### PR TITLE
Add net.sourceforge.molsketch

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:https://sourceforge.net/p/molsketch/feature-requests/31/**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build net.sourceforge.Molsketch

**Comment by hfiguiere** [comment]
apparently it can't find the appstream file

**Comment by kevinsmia1939** [comment]
Molsketch have trouble looking for libobabeliface.so, still working on solving the issue.

**Comment by kevinsmia1939** [comment]
bot, build net.sourceforge.Molsketch

**Comment by kevinsmia1939** [comment]
bot, build net.sourceforge.molsketch

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/net.sourceforge.molsketch.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
Validation of files failed
```

File: net.sourceforge.molsketch.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+command: molsketch
+rename-icon: molsketch
+rename-desktop-file: molsketch.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --filesystem=host
+  - --filesystem=home
```
Comment (hfiguiere) [inline net.sourceforge.molsketch.yaml:14]: one is redundant with the other.

**Comment by kevinsmia1939** [comment]
bot, build net.sourceforge.molsketch

**Comment by kevinsmia1939** [comment]
bot, build net.sourceforge.molsketch

**Comment by kevinsmia1939** [comment]
I think this is ready. Also please ping https://github.com/hvennekate if they want to be maintainer.

**Comment by kevinsmia1939** [comment]
bot, build net.sourceforge.molsketch

**Comment by barthalion** [comment]
@kevinsmia1939 Does upstream author have GitHub account so we can invite them?

**Comment by kevinsmia1939** [comment]
[hvennekate](https://github.com/hvennekate)
Would you like to be the maintainer as well? I usually do most of the flatpak maintenance work.

**Pull request closed without merging**